download_test_data.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+TEST_DATA_URL=${TEST_DATA_URL:-https://big-tank.app.tu-dortmund.de/lst-testdata/}
+
```

Comment:
Hi,
I would include in the test R0 data the 4 streams (few events per stream), so as to test the multi-file reading and to have proper drs4 time lapse corrections.

Reply:
Yes, I can produce that using the zfits recompressor. Any run particular?

Reply:
Done, there is now 

```
test_data/real/R0/20200218/LST-1.{1,2,3,4}.Run02008.0000_first50.fits.fz
test_data/real/R0/20200218/LST-1.{1,2,3,4}.Run02008.0100_first50.fits.fz
```

So the subruns 0 and 100 of run 2008 for all 4 streams truncated using `zfitsrecompressor` to the first 50 events for each stream.